src/comments.js

```diff
@@ -933,6 +938,38 @@ function printComment(commentPath, options) {
   }
 }
 
+function isJsDocComment(comment) {
+  const lines = comment.value.split("\n");
+  return (
+    lines.length > 1 &&
+    lines.slice(0, lines.length - 1).every(line => line.trim()[0] === "*")
+  );
+}
+
+function printJsDocComment(comment, options) {
+  const lines = comment.value.split("\n");
+  const tabWidth = options.tabWidth;
+  const indentSize = util.getIndentSize(comment.value, tabWidth);
+
+  const parts = [];
+
+  lines.forEach((line, index) => {
+    parts.push(
+      docBuilders.align(
```

Comment:
Why do we need to use `.align` rather than just  `(index > 0 ? " " : "") + line.trim()`?

Reply:
You're right, the `align` is not needed. Just a remnant of a previous approach to the problem.